### PR TITLE
Added a system property for skipping the sso server initialization

### PR DESCRIPTION
**Pull request by spisiakm**

### CODE REVIEW

**Comment by spisiakm** [comment]
@Ladicek let me know what you think

File: src/test/java/io/openshift/booster/SecuredBoosterIT.java [hunk 1]
```diff
@@ -46,7 +46,10 @@
 
   @BeforeClass
   public static void init() {
-    COMMAND_EXECUTOR.execCommand("oc create -f service.sso.yaml");
+    // You can disable the sso server initialization by setting the system property skip.sso.init to true
+    if (!Boolean.getBoolean("skip.sso.init")) {
```
[the remark this answers is not in the document]
Comment (spisiakm) [reply src/test/java/io/openshift/booster/SecuredBoosterIT.java]: Ah right, sorry, forgot about that.

**Comment by spisiakm** [comment]
@Ladicek done. I'll let @cescoffier merge both.

**Comment by Ladicek** [comment]
Thanks @spisiakm!

**Comment by cescoffier** [comment]
Thanks!

**Comment by spisiakm** [comment]
FYI: a JIRA for this has been created - https://issues.jboss.org/browse/RHOARDOC-1281

**Pull request merged**